server/app.js

```diff
@@ -46,10 +46,13 @@ app.use( ( req, res, next ) => {
   next();
 } );
 
+
+
 app.use( "/auth", authRouter );
 app.use( "/users", userRouter );
 app.use( "/profile", profileRouter );
 app.use( "/availability", availabilityRouter );
+app.use("/conversation",conversationRouter);
```

Comment:
```suggestion
app.use("/conversations",conversationRouter);
```
Nit! - to stay consistent with REST API naming convention